### ISSUE TITLE
Failed to build in production

### ISSUE THREAD

**Issue by iashraful** [ivan-kleshnin/react-safe#1]
I have used like that, `<Safe.p><div>Hello, World!</div></Safe.p>`. And then, I'm facing to build in the production version. 
[See the Image](https://imgur.com/a/PmGQy)

**Comment by ivan-kleshnin** [comment]
Thanks for reporting. I'll take a look.

**Comment by iashraful** [comment]
@ivan-kleshnin Then, moved to `<h1 dangerouslySetInnerHTML={{__html: page.title}}></h1>` this syntax. And it works for me fine.

**Comment by ivan-kleshnin** [comment]
The problem stems from React using UglifyJS which still does not support a single bit of ES6 syntax. And they aren't going to, choosing to release an alt. `uglify-es`  instead.

React is sticking to basic `uglify-js`. There's an issue proposing to shift:
https://github.com/facebookincubator/create-react-app/issues/2108
but it's not done yet.

I can address this problem by adding a build step or by avoiding ES6 syntax.
As ES5 don't have even `for-of` loop the first seems to be the better option.

---

As for your example:

```html
<Safe.div><div>test</div></Safe.div>
<div dangerouslySetInnerHTML={{__html: <div>test2</div>}}></div>
```

Both does not work as expected (rendering `[object Object]`) because both wait for a string. 

The correct syntax is:

```html
<Safe.div>{"<div>test</div>"}</Safe.div>
<div dangerouslySetInnerHTML={{__html: "<div>test2</div>"}}></div>
```

Passing React objects as strings can be handled but it does not seem right because HTML is a string by definition (as is JSON btw.).

**Comment by iashraful** [comment]
@ivan-kleshnin Thanks for your nice explanation. :+1:

**Comment by ivan-kleshnin** [comment]
No problem :) 

Should be fixed in 1.1.0. `create-react-app` production build works for me now.
Pls. close this issue if you confirm.
